app/controllers/shelters_controller.rb

```diff
@@ -13,6 +13,7 @@ def add
 
   def show
     @shelter = Shelter.find(params[:id])
+    @reviews = Review.where(shelter_id: @shelter.id)
```

Comment:
Kinda curious if this lines needs to be here. I believe you get the same information from `@shelter.reviews`